packages/rum/src/domain/record/privacy.ts

```diff
@@ -187,10 +187,16 @@ export function serializeAttribute(
     }
   }
 
-  // Rebuild absolute URLs from relative (without using <base> tag)
   if (!attributeValue || typeof attributeValue !== 'string') {
     return attributeValue
   }
+
+  // Minimum Fix for customer.
+  if (attributeValue.length > 100_000 && attributeValue.slice(0, 5) === 'data:') {
```

Comment:
`100_000` should be a constant like `MAX_DATA_ATTRIBUTE_LENGTH`, and be used in the tests too instead of `100000` to be more explicit

Reply:
Good catch.